projects/compose/koin-compose/src/commonMain/kotlin/org/koin/compose/Inject.kt

```diff
@@ -33,18 +32,15 @@ import org.koin.core.scope.Scope
  *
  * @author Arnaud Giuliani
  */
+@OptIn(KoinInternalApi::class)
 @Composable
 inline fun <reified T> koinInject(
     qualifier: Qualifier? = null,
     scope: Scope = currentKoinScope(),
     noinline parameters: ParametersDefinition? = null,
 ): T {
-    // This will always refer to the latest parameters
-    val currentParameters by rememberUpdatedState(parameters)
-
-    return remember(qualifier, scope) {
-        scope.get(qualifier) {
-            currentParameters?.invoke() ?: emptyParametersHolder()
-        }
+    val params: ParametersHolder? = parameters?.invoke()
```

Comment:
This is incorrect as it will recreate parameters on every recomposition

Reply:
we don't have choice to have stable parametersHolder, else it's a function